pkg/webui/components/tabs/tab/index.js

```diff
@@ -81,13 +81,18 @@ class Tab extends React.PureComponent {
       [style.tabItemDisabled]: disabled,
     })
 
-    const Component = link ? NavLink : 'span'
+    // There is no support for disabled on anchors in html and hence in `react-router`. So, do
+    // not render the link component if the tab is disabled, but render regular tab item instead.
+    const canRenderLink = link && !disabled
+
+    const Component = canRenderLink ? NavLink : 'span'
     const props = {
       role: 'button',
```

Comment:
```suggestion
      disabled,
      role: 'button',
```

See my other comment.

Reply:
`NavLink` doesnt have the `disabled` prop